### ISSUE TITLE
What is the dataset for fine-tuning?

### ISSUE THREAD

**Issue by majinyu666** [imatge-upc/retrieval-2016-deepvision#16]
Thanks for your great work. I have a question to ask.
It seems that the datasets for fine-tuning (oxford/paris/ins2013) are also the datasets you perform query on? Am I right? I'm new in image retrieval and I not sure if it's a convention, but it seems to be unfair if the "training set" and the "test set" are the same.

**Comment by amaiasalvador** [comment]
Training object detectors for instance search has been explored in the literature for the TRECVid benchmark (e.g. see https://luannd.github.io/papers/MMSP2015.pdf). The detector is trained on query images only, which comes with a cost in terms of computational time (a detector must be trained for each new query object), but also boosts performance.
